app/views/sections/items/ItemProducerSizeView.scala.html

```diff
@@ -47,19 +48,43 @@
 
         @h2(messages("items.subHeading"), "govuk-caption-xl", hiddenContent = Some(messages("subHeading.hidden")))
 
-        @h1(messages("itemProducerSize.title", messages(s"itemProducerSize.${goodsType.code}"), startYear, endYear))
 
-        @p(){@messages("itemProducerSize.p")}
+        @if(showAlcoholProductionContent){
+            @h1(messages("itemProducerSize.heading.pureAlcohol", startYear, endYear))
+
+        }else{
+            @h1(messages("itemProducerSize.heading.goodsType", messages(s"itemProducerSize.${goodsType.code}"), startYear, endYear))
+        }
+
+        @p(){@messages("itemProducerSize.p1")}
+
+        @if(showAlcoholProductionContent){
+            @p(){@messages("itemProducerSize.p2.pureAlcohol")}
+        }else{
+            @p(){@messages("itemProducerSize.p2.goodsType")}
+        }
 
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(Text(messages("itemProducerSize.title", messages(s"itemProducerSize.${goodsType.code}"), startYear, endYear))).hidden
+                label = if(showAlcoholProductionContent){
+                    LabelViewModel(Text(messages("itemProducerSize.label.pureAlcohol"))).withCssClass("govuk-!-font-weight-bold")
+                } else {
+                    LabelViewModel(Text(messages("itemProducerSize.label.goodsType"))).withCssClass("govuk-!-font-weight-bold")
+                },
```

Comment:
As above, you could assign the `if` logic to a value in the view and then call it here which will prevent duplication:

```suggestion
                label = LabelViewModel(Text(messages(labelMessageKey))).withCssClass("govuk-!-font-weight-bold")
              
```

Reply:
```
labelMessageKey = @{
    @if(showAlcoholProductionContent) {
        "itemProducerSize.label.pureAlcohol"
    } else {
        "itemProducerSize.label.goodsType"
    }
}
```